spec/rails_helper.rb

```diff
@@ -75,3 +75,19 @@
 def open_structify(data)
   JSON.parse(data.to_json, object_class: DatedStruct)
 end
+
+def stub_call_to_get_json_schema
```

Comment:
I think it would be safer and more flexible to pass the schema name into this method:
```ruby
def stub_call_to_schema(schema_name)
  stub_request(:get, "http://localhost:3000/schemas/#{schema_name}.json")
end
```
Then to use it:
```ruby
stub_call_to_schema :assessment_request
```